src/components/organisms/SearchBar.vue

```diff
@@ -33,42 +104,119 @@ export default defineComponent({
 <template>
   <v-card
     color="blue-grey darken-4"
-    height="1200px"
+    height="900px"
   >
-    <ToolBar
-      color="blue-grey darken-2"
-      flat
-    />
     <v-card-title>
       Sample Project
+      <v-spacer />
+      <v-card-actions>
+        <v-btn
+          v-if="searchResults"
+          plain
+          right
+          color="blue"
+          @click="reveal=true"
+        >
+          Show Results
+        </v-btn>
+      </v-card-actions>
     </v-card-title>
-    <!-- Will be needed in second iteration -->
-    <!-- <TabToolBar
+    <v-expand-transition>
+      <template>
+        <v-card
```

Comment:
Would dividing the search bar and search results into separate components be cleaner? Or maybe a top level `Search.vue` that contains all of the data/refs and then two child components `SearchBar.vue` and `SearchResults.vue` that take the data as props and displays it accordingly.

Just thinking of ways to break this component up into smaller, more digestible pieces. If you don't think that would yield any improvements, I think this component should at least be renamed to something more generic like `Search.vue` since it contains more than just the Search Bar.

Reply:
I agree, created Results.vue component